app/controllers/comments_controller.rb

```diff
@@ -18,6 +18,13 @@ def edit
     @comment = Comment.find(params[:id])
   end
 
+  def update
+    @comment = Comment.find(params[:id])
+    if @comment.update(comment_params)
+      redirect_to @comment.article, notice: t(".success")
+    end
```

Comment:
Do you have a story to cover an unsuccessful comment edit?

Reply:
I appreciate the feedback.
I have just created one [here](https://trello.com/c/KDbqdkko)

Reply:
This has been addressed in #226